db/migrate/20180823031202_create_pieces.rb

```diff
@@ -0,0 +1,14 @@
+class CreatePieces < ActiveRecord::Migration[5.0]
+  def change
+    create_table :pieces do |t|
+      t.string :name
+      t.integer :game_id
+      t.integer :user_id
+      t.string :position
```

Comment:
I tried to go back and see Joe's comment on the original pull request, but as it's been deleted I need to ask here - was the recommendation to change to just one x and y coordinate pair, or to move to a single string? I am mainly asking as this new approach will shift my approach on the `is_obstructed?` method. I can definitely adapt but wanted to clarify!

Reply:
So we'll need to consider what the position will look like. Proper chess notation would be something like "A5" and "H1". Do we want the location to be saved as the chess notation or something like `postion_x = 1` and `position_y = 5`. Also consider that on the front end, each square will be a div with an id of something like "A5" `<div id="A5"></div>`. So it will be easier to reference the div element on the front end if the position is saved as the id name of the square.

Reply:
On the other hand, it may be easier to perform calculations on piece positions with `position_x` and `position_y`, like the the `is_obstructed?` method... Yeah, I'll just change it to `position_x` and `position_y` like Joe said.